docs/rest-server/API.md

```diff
@@ -2889,6 +2889,172 @@ Status: 500
 }
 ```
 
+### `GET /api/v2/jobs/:frameworkName/jobAttempts/healthz`
+
```

Comment:
Why need this?

Can we just check this by using PAI feature gate: job-history=true? Like this
[https://github.com/microsoft/pai/blob/35489463cb5df9c2da85c139c88d37d7afeffdef/src/fluentd/deploy/start.sh.template#L20](https://github.com/microsoft/pai/blob/35489463cb5df9c2da85c139c88d37d7afeffdef/src/fluentd/deploy/start.sh.template#L20)

Reply:
You mean add a new ENV flag about this? From my point of view, the health check PAI is more flexible than configs, because it might be a scenario that the flag is true however elastic search or other prerequisites did not work for some rare reasons. I do not want this scenario to affect the main system works.

Reply:
But if so, if healthz not work but the feature gate enabled, we should not just suppress it, we should alert it. Is alert also in this PR?

Reply:
Diff disable and unexpected unhealthy

Reply:
![image](https://user-images.githubusercontent.com/8444268/68746640-f747d480-0633-11ea-9135-34623f32fd9e.png)
![image](https://user-images.githubusercontent.com/8444268/68746685-147ca300-0634-11ea-9c93-27511e0af53d.png)
![image](https://user-images.githubusercontent.com/8444268/68746842-6291a680-0634-11ea-9a14-fd0862dbf11c.png)